framework/bin/d4j/d4j-info

```diff
@@ -75,12 +75,26 @@ my $PID = $cmd_opts{p};
 my $BID = $cmd_opts{b};
 
 my $project = Project::create_project($PID);
+my $project_dir = "$SCRIPT_DIR/projects/$PID";
+
 my $revision_id;
 my $revision_date;
 my $bug_report_id;
 my $bug_report_url;
+
 # Check version id
 if (defined $BID) {
+    if ( ! -e "${project_dir}/trigger_tests/${BID}" ) {
+        print("${PID}-${BID} is a non-existent bug\n");
```

Comment:
Can you use die or confess (for consistency)? I am leaning toward confess for most errors, which provides a decent trace for debugging. Putting a newline (`confess("My error message\n")`) will separate the error message from the trace.